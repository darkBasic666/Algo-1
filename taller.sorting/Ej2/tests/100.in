100 0
[(Ceballos;7),(Coronel F. Varela;19),(Col. Oficial Nº14;16),(Ibarlucea;38),(Tres Lomas;12),(Col. Barón;26),(Río Chico;2),(Medellín;36),(Las Rosas;15),(San Benito;28),(Yaví;33),(Coghlan;1),(Maipú;29),(Maquinista Gallini;33),(Pueblo Cazes;45),(Hinojal;18),(Villa Amancay;45),(Pdcia. Roque Sáenz Peña;1),(Pergamino;27),(Gato Colorado;46),(Tabay;4),(J.J. Castelli;20),(La Cruz;19),(San Antonio de Padua;37),(Los Toldos;48),(San Roque;27),(Silvio Pellico;49),(Capitán Solari;19),(Santo Tomás;23),(Pablo Nogués;38),(Monte Hermoso;36),(Col. Pastoril;40),(Algarrobo del Aguila;1),(Isla Verde;38),(Angastaco;1),(Palmar Grande;33),(San Francisco Solano;41),(Lucila del Mar;22),(Granadero Baigorria;3),(Naupa Huen;9),(Simoca;6),(Rodeitos;9),(Punta del Monte;37),(La Gallareta;40),(Amenábar;24),(Rinconada;46),(Isletas;21),(Santurce;26),(Los Altares;39),(Sosa;47),(Las Moscas;8),(San Antonio;41),(Pehuajó;43),(San Pedro;8),(El Timbó;14),(Florencio Varela;38),(Ricardone;50),(San José;36),(El Trapiche;13),(Epuyén;7),(Gdor. Echagüe;16),(Villa Yacanto;28),(Chilibroste;48),(Calilegua;45),(Nicanor E. Molinas;12),(Salsacate;39),(Valle de Anisacate;38),(Col. Carrasco;29),(Las Cañadas;42),(Cosquín;27),(Gral. Galarza;30),(Reconquista;14),(Villa San José;7),(Los Reartes;40),(San Andrés de Giles;21),(Bajada del Agrio;8),(Atahona;21),(Villa Rossi;13),(Los Chañaritos (C.E);41),(Paso Duarte;34),(Dique F. Ameghino;46),(Las Saladas;24),(San Bernardo (N.J.);26),(Hermoso Campo;24),(Lonquimay;29),(Col. Viraró;47),(Tafí del Valle;45),(Aldea Epulef;31),(San Miguel de 25;23),(San Ignacio;45),(Caviahué;1),(Ituzaingó;13),(La Trinidad;42),(Ciervo Petiso;4),(Rivadavia Banda Norte;34),(Valle Grande;31),(Caucete;28),(Gral. Manuel Belgrano;9),(Bell Ville;40),(La Lucila;41)]
