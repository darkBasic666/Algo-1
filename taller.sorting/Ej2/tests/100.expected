100 0
[(Algarrobo del Aguila;1),(Angastaco;1),(Caviahué;1),(Coghlan;1),(Pdcia. Roque Sáenz Peña;1),(Río Chico;2),(Granadero Baigorria;3),(Ciervo Petiso;4),(Tabay;4),(Simoca;6),(Ceballos;7),(Epuyén;7),(Villa San José;7),(Bajada del Agrio;8),(Las Moscas;8),(San Pedro;8),(Gral. Manuel Belgrano;9),(Naupa Huen;9),(Rodeitos;9),(Nicanor E. Molinas;12),(Tres Lomas;12),(El Trapiche;13),(Ituzaingó;13),(Villa Rossi;13),(El Timbó;14),(Reconquista;14),(Las Rosas;15),(Col. Oficial Nº14;16),(Gdor. Echagüe;16),(Hinojal;18),(Capitán Solari;19),(Coronel F. Varela;19),(La Cruz;19),(J.J. Castelli;20),(Atahona;21),(Isletas;21),(San Andrés de Giles;21),(Lucila del Mar;22),(San Miguel de 25;23),(Santo Tomás;23),(Amenábar;24),(Hermoso Campo;24),(Las Saladas;24),(Col. Barón;26),(San Bernardo (N.J.);26),(Santurce;26),(Cosquín;27),(Pergamino;27),(San Roque;27),(Caucete;28),(San Benito;28),(Villa Yacanto;28),(Col. Carrasco;29),(Lonquimay;29),(Maipú;29),(Gral. Galarza;30),(Aldea Epulef;31),(Valle Grande;31),(Maquinista Gallini;33),(Palmar Grande;33),(Yaví;33),(Paso Duarte;34),(Rivadavia Banda Norte;34),(Medellín;36),(Monte Hermoso;36),(San José;36),(Punta del Monte;37),(San Antonio de Padua;37),(Florencio Varela;38),(Ibarlucea;38),(Isla Verde;38),(Pablo Nogués;38),(Valle de Anisacate;38),(Los Altares;39),(Salsacate;39),(Bell Ville;40),(Col. Pastoril;40),(La Gallareta;40),(Los Reartes;40),(La Lucila;41),(Los Chañaritos (C.E);41),(San Antonio;41),(San Francisco Solano;41),(La Trinidad;42),(Las Cañadas;42),(Pehuajó;43),(Calilegua;45),(Pueblo Cazes;45),(San Ignacio;45),(Tafí del Valle;45),(Villa Amancay;45),(Dique F. Ameghino;46),(Gato Colorado;46),(Rinconada;46),(Col. Viraró;47),(Sosa;47),(Chilibroste;48),(Los Toldos;48),(Silvio Pellico;49),(Ricardone;50)]
